_sass/components/_already-have-an-account.scss

```diff
@@ -0,0 +1,33 @@
+.one-account-banner__backdrop {
+    height: 22rem;
+
+    &::before {
+        background-image: none;
```

Comment:
I think this is unintentionally causing other instances of the banner to not have a background anymore, like the one on the "Who uses login.gov?" page:

![image](https://user-images.githubusercontent.com/1779930/121184445-179d3c80-c833-11eb-935a-326091f1f539.png)

If we're not intending to do that, and if these styles should be specific to the new banner only, maybe we could add a modifier class specific to that variation, so the styles only apply to it.

```html
<!-- _includes/already_have_an_account_banner.html -->
<aside class="one-account-banner already-have-an-account-banner">
```

```scss
// _sass/components/_already-have-an-account.scss
.one-account-banner.already-have-an-account-banner .one-account-banner__backdrop {
```